hebiros/WaypointMsg[] waypoints
float64[] times
---
sensor_msgs/JointState final_state
---
float64 percent_complete

